### DISCUSSION TITLE
True Casing in spaCy?

### DISCUSSION THREAD

**Discussion by SDAravind** [explosion/spaCy#8008] [New Features & Project Ideas]
TRUE CASING
---
Everybody knows text data is messy and one of them is incorrect **casing** in the text.

I was wondering if spaCy can include "**True Casing**" as new feature which solves NER problems and that can increase accuracy.

Below are few links that gives a heads up, looking for ways to solve this.

[True Casing in NLP](https://towardsdatascience.com/truecasing-in-natural-language-processing-12c4df086c21)

**PAPERS**
---
1. [tRuEcasIng](https://www.cs.cmu.edu/~llita/papers/lita.truecasing-acl2003.pdf)
2. [Learning to Capitalize with Character-Level Recurrent NeuralNetworks](https://www.aclweb.org/anthology/attachments/D16-1225.Attachment.pdf)

**Comment by polm** [comment]
I don't think we have any plans to add that as a core feature, but it'd be a welcome addition to the Universe!
